modules/machineset-creating.adoc

```diff
@@ -48,17 +48,21 @@ metadata:
   creationTimestamp: 2019-02-15T16:32:56Z <1>
   generation: 1 <1>
   labels:
+<<<<<<< HEAD
     machine.openshift.io/cluster-api-cluster: <cluster_name> <2>
+=======
+    machine.openshift.io/cluster-api-cluster:  <infrastructure_id> <2>
+>>>>>>> bug 1689241 updating the machineset definition
```

Comment:
Are these conflict markers intentional, or did they sneak in during a rebase?

Reply:
Thank you! https://github.com/openshift/openshift-docs/pull/14593